README.md

```diff
@@ -37,6 +37,7 @@ config := bigcache.Config{
 		MaxEntriesInWindow: 1000 * 10 * 60, // rps * lifeWindow
 		MaxEntrySize: 500,                  // max entry size in bytes, used only in initial memory allocation
 		Verbose: true,                      // prints information about additional memory allocation
+		HardMaxCacheSize: 8192,             // cache will not allocate more memory than this limit, value in MB, 0 means no limit
```

Comment:
If cache will not allocate more memory than limit what will happen? It should be described here. I think `Hard` prefix is not needed here